grid/unit_test/tstFastFourierTransform.hpp

```diff
@@ -39,7 +39,8 @@ void calculateFFT( bool use_default, bool use_params,
     Experimental::FastFourierTransformParams params;
 
     // Set MPI communication
-    params.setAllToAll( true );
+    params.setAlltoAll( Cabana::Grid::Experimental::FFTCommPattern::alltoallv );
+    params.setAlltoAll( true );
```

Comment:
Do we need to set with both?

Reply:
You are right, only one is needed. I will remove the boolean variation. Its behavior and usage is commented at the example [example/grid_tutorial/10_fft_heffte/heffte_fast_fourier_transform_example.cpp](https://github.com/ECP-copa/Cabana/pull/799/files#diff-0ed0f7a30244d6118a480989c9185939d650f85b0c619c95ec62719d57664458) for backward compatibility